docs/Documentation/HowToAddNewOutputVars.md

```diff
@@ -1,14 +1,14 @@
 # How to Add New Output Variables
 
-VIC allows users to select which output variables to write to their output files. Variables that can be selected are listed [here](OutputVarList.md) as well as in the VIC source code file `vic_def.h` (listed as `OUT_*`). However, not all variables that VIC simulates are available for output. And of course if you are changing VIC's physics, any new variables you add are also not available for output (yet).
+VIC allows users to select which output variables to write to their output files. Variables that can be selected are listed [here](OutputVarList.md) as well as in the VIC source code file `vic_driver_shared_all.h` (listed as `OUT_*`). However, not all variables that VIC simulates are available for output. And of course if you are changing VIC's physics, any new variables you add are also not available for output (yet).
 
 You can enable VIC to output new variables by making a few changes to the code, as follows:
 
-## 1\. Make sure VIC sends the variable to put_data()
+## 1. Make sure VIC sends the variable to put_data()
 
-If VIC doesn't already compute the variable you need, or computes it but doesn't store it in one of the structures that is accessible to put_data(), you'll have to modify VIC to do this. The structures that put_data() has access to are: dmy, atmos_data, soil_con, cell_data, veg_con, veg_var, energy, snow_data, and all_vars.
+If VIC doesn't already compute the variable you need, or computes it but doesn't store it in one of the structures that is accessible to put_data(), you'll have to modify VIC to do this. The structures that put_data() has access to are: atmos_data, soil_con, cell_data, veg_con, veg_var, energy, snow_data, and all_vars.
 
```

Comment:
This is a minor point - isn't `all_vars` just a collection of `cell_data`, `snow_data`, `veg_var` and `energy` (and lake)?